acceptance-tests/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
   testImplementation project(':consensus:ibft')
   testImplementation project(':crypto')
   testImplementation project(':enclave')
+  testImplementation project(':ethsigner')
```

Comment:
My thinking here was that the `EthSignerClient` would be used for signing the marker transaction - but sure for the purpose of this PR it can live in the dsl.